### PR TITLE
Update issue_template

### PR DESCRIPTION
**Pull request by VictorAlagwu**

### CODE REVIEW

File: pull-request-template.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+---------------------------
+**Operating System:** …
+
+**Platform:** …
+
+**Target Platform:** …
+
+**Toolchain Version:** …
+
+**SDK Version:** …
```
Comment (SD10) [inline pull-request-template.md]: Can we change this to "MessageKit Version:" ?

File: pull-request-template.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Where has this been tested?
+---------------------------
+**Operating System:** …
+
+**Platform:** …
+
+**Target Platform:** …
+
+**Toolchain Version:** …
```
Comment (SD10) [inline pull-request-template.md]: "Swift Version:"

File: pull-request-template.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+-------------------------------------
+(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)
+
+Any other comments?
+-------------------
+…
+
+Where has this been tested?
+---------------------------
+**Operating System:** …
```
Comment (SD10) [inline pull-request-template.md]: "iOS Version:"

File: pull-request-template.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+**Platform:** …
+
+**Target Platform:** …
```
Comment (SD10) [inline pull-request-template.md]: "Devices/Simulators:"

File: pull-request-template.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+Any other comments?
+-------------------
+…
+
+Where has this been tested?
+---------------------------
+**Operating System:** …
+
+**Platform:** …
```
Comment (SD10) [inline pull-request-template.md]: Don't need this one

**Review by SD10 — CHANGES_REQUESTED** [review]
@VictorAlagwu Good work. Just a few changes to make it more specific to our project. No need for a new PR. Just make the changes with another commit.

**Comment by VictorAlagwu** [comment]
I have done that

**Review by Sherlouk — COMMENTED** [review]
Could we also move these to the `.github` directory as to not clog up the root directory? 

Also I have mild OCPD so can we name the files consistently: `issue_template.md` (same) and `pull_request_template.md` (underscores, not hypens).

**Comment by VictorAlagwu** [comment]
Will do that

**Comment by VictorAlagwu** [comment]
Done

**Review by SD10 — APPROVED** [review]

**Comment by SD10** [comment]
Thank you @VictorAlagwu 🎉 👍

**Comment by VictorAlagwu** [comment]
You are welcome

**Pull request merged**